### ISSUE TITLE
Non-static method pts_test_run_manager::set_batch_mode() should not be called statically in s tress_run:44

### ISSUE THREAD

**Issue by sandalle** [phoronix-test-suite/phoronix-test-suite#114]
Issue first found at http://www.phoronix.com/forums/forum/phoronix/phoronix-test-suite/854665-stress-run-not-working

I'm seeing this same issue with Phoronix Test Suite v6.2.1 (latest stable release) on RHEL 6.7 with php 5.3.3-40.el6_6.

### Error
```
[root@host ~]# TOTAL_LOOP_TIME=480 PTS_CONCURRENT_TEST_RUNS=3 phoronix-test-suite stress-run hpcc build-linux-kernel c-ray fio encode-flac

Network Communication Is Disabled Per Your User Configuration.


[2048] Non-static method pts_test_run_manager::set_batch_mode() should not be called statically in s
tress_run:44
PHP Fatal error:  Using $this when not in object context in /usr/share/phoronix-test-suite/pts-core/
objects/client/pts_test_run_manager.php on line 1004
```

### System Information
```
[root@host ~]# phoronix-test-suite system-info

Network Communication Is Disabled Per Your User Configuration.


Phoronix Test Suite v6.2.1
System Information

Hardware:
Processor: 2 x Intel Xeon E5-2699 v3 @ 2.30GHz (72 Cores), Motherboard: HP ProLiant DL380 Gen9, Chip
set: Intel Xeon E7 v3/Xeon, Memory: 16 x 16384 MB DDR4-2133MHz, Disk: 900GB LOGICAL VOLUME, Graphics
: Matrox s MGA G200EH, Network: Broadcom NetXtreme BCM5719 Gigabit PCIe

Software:
OS: Red Hat Enterprise Linux Server 6.7, Kernel: 2.6.32-573.7.1.el6.x86_64 (x86_64), Compiler: GCC 4
.4.7 20120313, File-System: ext4
```

```
[root@host ~]# rpm -qa php* | sort
php-5.3.3-40.el6_6.x86_64
php-cli-5.3.3-40.el6_6.x86_64
php-common-5.3.3-40.el6_6.x86_64
php-gd-5.3.3-40.el6_6.x86_64
php-pdo-5.3.3-40.el6_6.x86_64
php-xml-5.3.3-40.el6_6.x86_64
```

I also get this error with the latest version from GitHub

```
[root@host phoronix-test-suite]# ./phoronix-test-suite version

Network Communication Is Disabled Per Your User Configuration.


Phoronix Test Suite v6.2.1 (Gamvik)
```

```
[root@host phoronix-test-suite]# git log | head
commit 54e667ed2ba40856cc98d40cab3264a5964717f0
Merge: 2a730dc ce1acaa
Author: Michael Larabel <michaellarabel@users.noreply.github.com>
Date:   Sun Feb 28 07:49:53 2016 -0600

    Merge pull request #112 from atom-symbol/master

    Remove superfluous quote from phoromatic.css

commit ce1acaa6fb13f15897d1970466ea980ad10cac54
```

**Comment by michaellarabel** [comment]
Hi, if you pull from Git, is your problem now resolved?

Thanks,
MIchael

**Comment by sandalle** [comment]
Yes, commit 3fad54cefe2dfb5c43a9b579f3050706eb7dfeab appears to have fixed it, thank you! :)

**Comment by eusonlito** [comment]
Hi, same here.

Installed from deb package downloaded on Phoronix website `phoronix-test-suite_6.4.0_all.deb`.

Using 

```
# uname -a
Linux Ubuntu-1604-xenial-64-minimal 4.4.0-22-generic #40-Ubuntu SMP Thu May 12 22:03:46 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

```
    pts/phpbench-1.1.0:
        Test Installation 6 of 10
        1 File Needed [0.01 MB / 1 Minute]
        Downloading: phpbench-0.8.1.tar.gz                                                                                                                                                      [0.01MB]
        Estimated Download Time: 1m ....................................................................................................................................................................
        Installation Size: 1 MB
        Installing Test @ 20:19:27
```

```
# phoronix-test-suite run-random-tests

Phoronix Test Suite v6.4.0
Random Test Execution

    Allow new tests to be installed (Y/n): 
    Allow new test external dependencies to be installed (y/N): Y
    Limit tests to a given subsystem (y/N): 
    Auto-upload test results to OpenBenchmarking.org (Y/n): 

    Tests To Run: pts/openssl-1.9.0, pts/cuda-mini-nbody-1.0.1, pts/timed-audio-encode-1.0.0
    Installed: pts/openssl-1.9.0
    Installed: pts/cuda-mini-nbody-1.0.1
    Installed: pts/timed-audio-encode-1.0.0

[8192] Non-static method pts_test_run_manager::set_batch_mode() should not be called statically in run_random_tests:137
PHP Fatal error:  Uncaught Error: Using $this when not in object context in /usr/share/phoronix-test-suite/pts-core/objects/client/pts_test_run_manager.php:1014
Stack trace:
#0 /usr/share/phoronix-test-suite/pts-core/commands/run_random_tests.php(137): pts_test_run_manager::set_batch_mode(Array)
#1 /usr/share/phoronix-test-suite/pts-core/objects/client/pts_client.php(1226): run_random_tests::run(Array)
#2 /usr/share/phoronix-test-suite/pts-core/phoronix-test-suite.php(173): pts_client::execute_command('run_random_test...', Array)
#3 {main}
  thrown in /usr/share/phoronix-test-suite/pts-core/objects/client/pts_test_run_manager.php on line 1014
```

```
# phoronix-test-suite system-info

Phoronix Test Suite v6.4.0
System Information

Hardware:
Processor: Intel Xeon E5-26xx @ 2.10GHz (1 Core), Motherboard: Hetzner vServer v2, Chipset: Intel 440FX- 82441FX PMC, Memory: 1024MB, Disk: 26GB QEMU HDD, Graphics: Device 1234:1111, Network: Red Hat Virtio device

Software:
OS: Ubuntu 16.04, Kernel: 4.4.0-22-generic (x86_64), Compiler: GCC 5.4.0 20160609, File-System: ext4, System Layer: KVM
```

```
# dpkg -l php* | grep ii
ii  php-common            1:35ubuntu6                 all          Common files for PHP packages
ii  php-pear              1:1.10.1+submodules+notgz-6 all          PEAR Base System
ii  php7.0-bcmath         7.0.8-0ubuntu0.16.04.2      amd64        Bcmath module for PHP
ii  php7.0-bz2            7.0.8-0ubuntu0.16.04.2      amd64        bzip2 module for PHP
ii  php7.0-cgi            7.0.8-0ubuntu0.16.04.2      amd64        server-side, HTML-embedded scripting language (CGI binary)
ii  php7.0-cli            7.0.8-0ubuntu0.16.04.2      amd64        command-line interpreter for the PHP scripting language
ii  php7.0-common         7.0.8-0ubuntu0.16.04.2      amd64        documentation, examples and common module for PHP
ii  php7.0-curl           7.0.8-0ubuntu0.16.04.2      amd64        CURL module for PHP
ii  php7.0-dev            7.0.8-0ubuntu0.16.04.2      amd64        Files for PHP7.0 module development
ii  php7.0-enchant        7.0.8-0ubuntu0.16.04.2      amd64        Enchant module for PHP
ii  php7.0-fpm            7.0.8-0ubuntu0.16.04.2      amd64        server-side, HTML-embedded scripting language (FPM-CGI binary)
ii  php7.0-gd             7.0.8-0ubuntu0.16.04.2      amd64        GD module for PHP
ii  php7.0-gmp            7.0.8-0ubuntu0.16.04.2      amd64        GMP module for PHP
ii  php7.0-imap           7.0.8-0ubuntu0.16.04.2      amd64        IMAP module for PHP
ii  php7.0-interbase      7.0.8-0ubuntu0.16.04.2      amd64        Interbase module for PHP
ii  php7.0-intl           7.0.8-0ubuntu0.16.04.2      amd64        Internationalisation module for PHP
ii  php7.0-json           7.0.8-0ubuntu0.16.04.2      amd64        JSON module for PHP
ii  php7.0-ldap           7.0.8-0ubuntu0.16.04.2      amd64        LDAP module for PHP
ii  php7.0-mbstring       7.0.8-0ubuntu0.16.04.2      amd64        MBSTRING module for PHP
ii  php7.0-mcrypt         7.0.8-0ubuntu0.16.04.2      amd64        libmcrypt module for PHP
ii  php7.0-mysql          7.0.8-0ubuntu0.16.04.2      amd64        MySQL module for PHP
ii  php7.0-odbc           7.0.8-0ubuntu0.16.04.2      amd64        ODBC module for PHP
ii  php7.0-opcache        7.0.8-0ubuntu0.16.04.2      amd64        Zend OpCache module for PHP
ii  php7.0-pgsql          7.0.8-0ubuntu0.16.04.2      amd64        PostgreSQL module for PHP
ii  php7.0-phpdbg         7.0.8-0ubuntu0.16.04.2      amd64        server-side, HTML-embedded scripting language (PHPDBG binary)
ii  php7.0-pspell         7.0.8-0ubuntu0.16.04.2      amd64        pspell module for PHP
ii  php7.0-readline       7.0.8-0ubuntu0.16.04.2      amd64        readline module for PHP
ii  php7.0-recode         7.0.8-0ubuntu0.16.04.2      amd64        recode module for PHP
ii  php7.0-soap           7.0.8-0ubuntu0.16.04.2      amd64        SOAP module for PHP
ii  php7.0-sqlite3        7.0.8-0ubuntu0.16.04.2      amd64        SQLite3 module for PHP
ii  php7.0-sybase         7.0.8-0ubuntu0.16.04.2      amd64        Sybase module for PHP
ii  php7.0-tidy           7.0.8-0ubuntu0.16.04.2      amd64        tidy module for PHP
ii  php7.0-xml            7.0.8-0ubuntu0.16.04.2      amd64        DOM, SimpleXML, WDDX, XML, and XSL module for PHP
ii  php7.0-xmlrpc         7.0.8-0ubuntu0.16.04.2      amd64        XMLRPC-EPI module for PHP
ii  php7.0-xsl            7.0.8-0ubuntu0.16.04.2      all          XSL module for PHP (dummy)
ii  php7.0-zip            7.0.8-0ubuntu0.16.04.2      amd64        Zip module for PHP
```

Regards.

**Comment by michaellarabel** [comment]
@eusonlito the issue is fixed if using Git or the 6.6.0 release will be out in about one week.
